src/Features/Core/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer_GetDiagnostics.cs

```diff
@@ -312,7 +332,7 @@ public IDECachedDiagnosticGetter(DiagnosticIncrementalAnalyzer owner) : base(own
 
                     foreach (var document in project.Documents)
                     {
-                        await AppendProjectAndDocumentDiagnosticsAsync(state, project, predicate, cancellationToken).ConfigureAwait(false);
+                        await AppendProjectAndDocumentDiagnosticsAsync(state, document, predicate, cancellationToken).ConfigureAwait(false);
```

Comment:
I thought @shyamnamboodiripad fixed this. you might have conflict with @shyamnamboodiripad change.